synapse/lib/node.py

```diff
@@ -60,6 +60,9 @@ def getByLayer(self):
         '''
         return s_msgpack.deepcopy(self.bylayer)
 
+    def __str__(self):
+        return str(self.ndef[1])
```

Comment:
Can we make this include the form name too? Maybe something like:
`return f'{ndef[0]}={ndef[1]}`

Mostly because if I'm printing nodes in a pipeline of mixed forms, seeing idens roll by isn't very useful.